mcstatus/bedrock_status.py

```diff
@@ -8,6 +8,10 @@
 import asyncio_dgram
 
 from mcstatus.address import Address
+from mcstatus.status_response import BedrockStatusResponse
+
+# TODO Remove this useless __all__ after 2022-08
```

Comment:
Because it does nothing after the deprecation period. This variable exists here, to annotate for type checker that this module re-export the variable.

```py
# some.py
from bazz import fuzz
# other.py
from some import fuzz  # oops, error. `some.py` doesn't explicitly reexport this variable. 

# some.py
from bazz import fuzz

__all__ = ["fuzz"]
# other.py
from some import fuzz  # no error.
```
After deprecation period, we have to remove `BedrockStatusResponse` from `__all__` there and in that way this `__all__` will be equal to auto generated `__all__`.

I can rephrase it, as it's quite confusing.

Reply:
Actually, in your example, I'm pretty sure the import would work. The reason we use `__all__` is merely to alter how wildcard imports (`from some import *`) act. With the wildcard import, indeed, `fuzz` wouldn't get imported in the first example, but would if it's defined in `__all__`. 

However explicit imports will always work, no matter whether those names are stated in `__all__` or not, as long as they're defined, explicit imports will bring those items into scope.

Reply:
You're right with pyright, which is strange for me. But with my config and mypy it wouldn't
https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-no-implicit-reexport

Reply:
Well, I'm talking about pure python, no type checkers, what type checkers do about those imports is a different question, and indeed, mypy checks imports for presence in `__all__` (or using the default strategy if it's not defined), pyright doesn't. That's however not related to real functionality, and pyright is sort of right to not check for this presence, as that's how python would work on runtime.